pkg/console/operator/sync_v400.go

```diff
@@ -169,12 +167,14 @@ func sync_v400(co *consoleOperator, operatorConfig *operatorv1.Console, consoleC
 		return err
 	}
 
-	if _, err := SyncConsoleConfig(co, consoleConfig, consoleURL); err != nil {
+	// TODO: we are mutating the original here.  Thats not great.  We should copy
+	// and update, like we do with the operator config.
```

Comment:
Do we want to fix that here or as a followUp ?

Reply:
I'll do a `console.DeepCopy()` inside that func right before the `.UpdateStatus()` call.  That should do the trick.